src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java

```diff
@@ -66,7 +66,7 @@ public static String getCaseID(SscsCaseData caseData) {
     }
 
     public static String getCaseDeepLink(SscsCaseData caseData) {
-        return String.format("%s/cases/case-details/%s", exUiUrl, getCaseID(caseData));
+        return String.format("%s%s%s", exUiUrl, CASE_DETAILS_URL, getCaseID(caseData));
```

Comment:
Minor: Can we change this to the way it was in the other PR? Using the template string instead?

Reply:
Sorry, I pushed to this one by mistake, ill remove this now